maven-settings.xml

```diff
@@ -2,7 +2,7 @@
   <mirrors>
     <mirror>
       <id>repo.jenkins-ci.org</id>
-      <url>https://repo.azure.jenkins.io/public/</url> <!-- INFRA-1176 -->
+      <url>https://repo.jenkins-ci.org/public/</url>
```

Comment:
on ci.jenkins this should use the version pulled from by retrieveMavenSettingsFile, but everyone else/fallback will use artifactory instead of the dead url